administration/aws-credentials.md

```diff
@@ -0,0 +1,36 @@
+# AWS Credentials
+
+Plugins that interact with AWS services will fetch credentials from various providers in the following order.
+Only the first provider that is able to provide credentials will be used.
```

Comment:
May be add a sentence saying that you can additionally use credentials from an additional IAM Role via the `role_arn`/`aws_role_arn` fields in all AWS plugins. However, what is shown here is the standard credential provider chain and it is used as the base creds to assume that role.

Reply:
Added a note for this. Also it seems that GitHub did not revoke your approval when I pushed.